packages/vite/src/node/ssr/ssrExternal.ts

```diff
@@ -157,7 +157,9 @@ function collectExternals(
       }
       // check if the entry is cjs
       const content = fs.readFileSync(esmEntry, 'utf-8')
-      if (/\bmodule\.exports\b|\bexports[.\[]|\brequire\s*\(/.test(content)) {
+      const CJS_CONTENT_RE =
```

Comment:
**suggestion:** We could move this constant out of the function scope

Reply:
done